.github/workflows/swaggerhub-delete.yml

```diff
@@ -0,0 +1,26 @@
+name: Delete spec version from SwaggerHub
+
+on:
+  [delete]
+
+jobs:
+  removespec:
+    name: Remove the corresponding spec from SwaggerHub
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: set output for versioning
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+
+      - name: remove spec from swaggerhub
+        env:
+          AUTH_TOKEN: ${{secrets.SGH_TOKEN}}
```

Comment:
May we name it `SWAGGERHUB_TOKEN`? As for me it will be more clear for anyone who will read action file and configure secrets.